src/modules/ekf2/ekf2_main.cpp

```diff
@@ -263,79 +283,158 @@ Ekf2::Ekf2():
 	_lp_roll_rate(250.0f, 30.0f),
 	_lp_pitch_rate(250.0f, 30.0f),
 	_lp_yaw_rate(250.0f, 20.0f),
-	_ekf(new Ekf()),
```

Comment:
@tumbili What is the problem with the structure here? This is completely valid C++